src/components/home.vue

```diff
@@ -48,7 +48,7 @@ export default {
     fetchData() {
       this.$store.dispatch('get', [{
         key: 'projects',
-        url: '/v1/projects',
+        url: '/v1/projects-forms', // todo: should this be using apiPaths?
```

Comment:
Just thought I'd comment on this todo, I usually don't use `apiPaths` if the URL is constant.